packages/react/src/components/Authenticator/shared/FormField.tsx

```diff
@@ -29,6 +30,8 @@ export function FormField({
     [name, validationErrors]
   );
   const hasError = errors?.length > 0;
+  const errorId = useStableId();
+  const ariaDescribedBy = hasError ? errorId : undefined;
```

Comment:
Do we currently allow customization of these fields (via the Authenticator) that would allow someone to pass through descriptiveText for their own initial aria-describedBy attribute? As we do for the [primitive PasswordField](https://ui.docs.amplify.aws/react/components/passwordfield#descriptive-text)? If not, this is fine. If yes, we would have to do some managing of the id reference list that aria-describedBy supports (you can have more than one ID for the aria-describedBy attribute on an element)

Reply:
We do not -- https://ui.docs.amplify.aws/react/components/authenticator#form-field-customization.

I'll go ahead with this approach then!